src/akeneo/application/skeleton/composer.json.twig

```diff
@@ -102,9 +102,14 @@
     "compatibility": "bin/phpcs -s --standard=vendor/phpcompatibility/php-compatibility/PHPCompatibility/ --runtime-set testVersion 7.3-"
   },
   "config": {
+    "allow-plugins": {
+      "dealerdirect/phpcodesniffer-composer-installer": true,
+      "symfony/flex": true
+    },
     "platform": {
       "php": "{{ (version_compare(@('akeneo.major_version'), '5', '>=')) ? '7.4' : '7.3.22' }}"
     }
   },
-  "minimum-stability": "stable"
+  "minimum-stability": "alpha",
```

Comment:
if you need alpha packages, put the @alpha on the package, don't allow all packages to be alpha